src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseConfirmsToProceedRespondentNotificationHandler.java

```diff
@@ -78,4 +104,9 @@ private boolean isCcNotification(CallbackParams callbackParams) {
         return callbackParams.getRequest().getEventId()
             .equals(NOTIFY_RESPONDENT_SOLICITOR1_FOR_CLAIMANT_CONFIRMS_TO_PROCEED_CC.name());
     }
+
+    private boolean isRespondentSolicitor2Notification(CallbackParams callbackParams) {
```

Comment:
Can we move this to a static class so as to not repeat it in two classes.

Reply:
I followed the same pattern as above method. the method is specific to check specific to this handler.